packages/workspace/src/command-line/workspace-schematic.ts

```diff
@@ -288,6 +320,10 @@ async function executeSchematic(
     if (nothingDone) {
       logger.info('Nothing to be done.');
     }
+
+    if (options.dryRun) {
+      this.logger.warn(`\nNOTE: The "dryRun" flag means no changes were made.`);
```

Comment:
`this.logger.warn` here vs simply `logger.info` above... is there a reason they're different?

Reply:
Using `logger.info` above allows custom colors to be used in the log message for only a subsection of the string if required.

It should be `logger.warn` without the `this`, I'll fix that. Using the `.warn` tells the `cli-logger` to change the color of the full log message to yellow, or to whichever color the `@angular/devkit/core/node` decides to potentially use in future.

Reply:
It was the usage of `this` that I was commenting on. I should have been clearer :)

Reply:
Ah ok. Yes, it was mistake. Good spot!

Reply:
Change implemented